base/sparse/csparse.jl

```diff
@@ -313,8 +313,17 @@ function csc_permute{Tv,Ti}(A::SparseMatrixCSC{Tv,Ti}, pinv::Vector{Ti}, q::Vect
     (C.').' # double transpose to order the columns
 end
 
+
 # based on cs_symperm p. 21, "Direct Methods for Sparse Linear Systems"
 # form A[p,p] for a symmetric A stored in the upper triangle
+doc"""
+    symperm(A, p)
+
+Return the symmetric permutation of `A`, which is `A[p,p]`. `A` should be
+symmetric and sparse, where only the upper triangular part of the matrix is
```

Comment:
More awkward wording? I'd say "`A` should be symmetric, sparse, and with the upper triangular part stored. Only the upper triangular part of `A` is returned."